addons/survey/views/survey_question_views.xml

```diff
@@ -94,6 +94,11 @@
                             </div>
                         </group>
                     </group>
+                    <div>
+                        <label for="description" string="Description"/> 
+                        <separator />
+                        <field name="description" widget="html"/>
+                    </div>
```

Comment:
I would put that description in its own tab.
"Answers", then "Description", then "Options".

It takes a lot of space and will be left blank most of the time so it can be put in its own tab.

Reply:
done